lib/mongoid/extensions/stringified_symbol.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+module Mongoid
+  module Extensions
+    module StringifiedSymbol
```

Comment:
I suggest moving this class directly into `Mongoid` module, "extensions" generally refers to modifying behavior of core/standard library classes.